README.md

```diff
@@ -75,17 +75,17 @@ Therefore I created this script to allow you to easily scan for and purge these
+If you feel sketched out about giving the app the required high level permissions to your channel (very understandable), you could instead use the app in 'moderator mode' (set in the config file). First, some context: When you grant access to another channel to be a moderator for your channel, they are able to mark comments as 'held for review', and this permission works through the API as well. 
 
-_Therefore,_ what you could do is create an _blank dummy-google-account_ with nothing on it except a empty new channel. Then you can grant _that_ channel permission to be a moderator, and use the app through _the dummy moderator account_. **This way, you know that the app will never have the ability to do more than mark comments as held for review** (which the app supports) on your main channel, and have no other access to your account's data. You just won't be able to ban the spammers through this app directly, but you can still remove/hide their comments instead of deleting them. Just make sure to create the google cloud API project on the dummy account instead.
+_Therefore,_ what you could do is create a _blank dummy-google-account_ with nothing on it except an empty new channel. Then you can grant _that_ channel permission to be a moderator, and use the app through _the dummy moderator account_. **This way, you know that the app will never have the ability to do more than mark comments as held for review** (which the app supports) on your main channel, and have no other access to your account's data. You just won't be able to ban the spammers through this app directly, but you can still remove/hide their comments instead of deleting them. Just make sure to create the google cloud API project on the dummy account instead.
 
 Read some additional details about 'moderator mode' on the [wiki page here][Moderator Mode].
 
 ## Usage Notes -READ THIS
 
 1. To use this script, you will need to obtain your own API credentials file by making a project via the Google Developers Console (aka 'Google Cloud Platform'). The credential file should be re-named `client_secret.json` and be placed in the same directory as this script. [See Instructions Here][API Key].
 
-2. **IF IT FREEZES** while scanning, it is probably because you clicked within the command prompt window and entered "selection mode" which pauses everything. **To unfreeze it, simply right click within the window, or press the Escape key.**
+2. **IF IT FREEZES** while scanning, it is probably because you clicked within the command prompt window and entered "selection mode" which pauses everything. **To unfreeze it, simply-right click within the window, or press the Escape key.**
```

Comment:
Is this a grammatical error?

Reply:
This is what I was referring to, but for some reason I said “click” instead of “right”.
`simply-right` should not be compounded, they should be separate words, as in the original `simply right`.
If I were to compound any part of that, it would be `right-click` due to it being a single action.

Reply:
So this is how this it is meant to be?

Reply:
This is the error, BTW that you need to fix.

Reply:
This looks like a mistake. In my knowledge of English, there shouldn't be a-dash there.